client/src/components/content/ModelerSidebar/ModelerSidebar.jsx

```diff
@@ -106,16 +106,16 @@ const ModelerSidebar = ({ modeler, robotId }) => {
             disableTaskSelection={disableTaskSelection}
             element={elementState.currentElement}
             robotId={robotId}
-            variableList={variableList}
+            parameterList={parameterList}
             parameterSelectionUpdated={(newValue) => {
               inputParameterChangeHandler(
                 elementState.currentElement.id,
                 newValue
               );
             }}
-            outputVariableName={outputVariableName}
+            outputValueName={outputValueName}
             outputNameUpdated={(newValue) => {
-              outputVarNameChangeHandler(
+              outputValNameChangeHandler(
```

Comment:
same with the short form here
```suggestion
              outputValueNameChangeHandler(
```